src/utils.js

```diff
@@ -13,18 +13,16 @@ const mkdirp = Promise.promisify(mkdir);
 const writeFile = Promise.promisify(fs.writeFile);
 
 /**
- * Parse a (`.zel`) file's base64 string as JSON.
- * Customizes handler for `Unexpected Token` error
+ * Converts a Buffer (base64 string) to a JSON object.
  *
- * @param Buffer} content - The file's contents
- * @param {string} filepath - The file's path
+ * @param {Buffer} content - The Buffer to convert
  * @return {Object}
  */
-export function bufferToJSON(content: Buffer, filepath: string): Object {
+export function bufferToJSON(content: Buffer): Object {
     try {
         return JSON.parse(Buffer.from(content, 'base64').toString('utf8'));
     } catch (err) {
-        throw `Unexpected token in ${filepath}`;
+        throw err;
```

Comment:
Hm, the problem is that `bufferToJSON` will now be tied to an optional `filepath` instead of a truly generic function should we need to reuse it in the future for another case. I guess for now we can just include it and change it when it comes up.